src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1575,7 +1575,7 @@ CHIP_ERROR CASESession::HandleSigma3a(System::PacketBufferHandle && msg)
     TLV::TLVType containerType = TLV::kTLVType_Structure;
 
     const uint8_t * buf   = msg->Start();
-    const uint16_t bufLen = msg->DataLength();
+    const uint32_t bufLen = static_cast<uint32_t>(msg->DataLength());
```

Comment:
Can bufLen not be size_t?

Reply:
This one can be size_t without any other perturbations I think. Will change.